docs/js/example.js

```diff
@@ -1,14 +1,125 @@
-var x = new XMLHttpRequest();
+/* global VANILLA_VERSION */
+(function() {
+  if (!window.VANILLA_VERSION) {
+    throw Error('VANILLA_VERSION not specified.');
```

Comment:
@anthonydillon One of the not ideal solutions used is the need for latest Vanilla version. I pass it via global variable (which seems the easiest and most cross browser friendly).

Optionally we could:
-  just hardcode it in URL in the script, but that would be one more place to update on releases
- pass it via attribute on `<script src="example.js" data-version="2.5.0">` and access via `document.currentScript` - but that would require polyfill for older browsers

Any other ideas?

Reply:
I think this solution is better than the two hardcoded versions. The less we have to change per release the better.